centreon/www/front_src/src/CloudNotificationsConfiguration/EditPanel/ReducePanel.tsx

```diff
@@ -0,0 +1,41 @@
+import { useTranslation } from 'react-i18next';
+import { useAtom } from 'jotai';
+import { gt } from 'ramda';
+
+import { Box, Button } from '@mui/material';
+
+import {
+  labelExpandInformationPanel,
+  labelReduceInformationPanel
+} from '../translatedLabels';
+import { panelWidthStorageAtom } from '../atom';
+
+import useStyles from './Form.styles';
+
+const ReducePanel = (): JSX.Element => {
+  const { classes } = useStyles();
+  const { t } = useTranslation();
+  const [panelWidth, setPanelWidth] = useAtom(panelWidthStorageAtom);
+
+  const handlePanelWidth = (): void => {
+    setPanelWidth((prevState) => (gt(prevState, 675) ? 550 : 800));
```

Comment:
To be defined: I think those values should be defined in the utils folder in Centreon UI